src/main/java/com/mail/entities/TextMessage.java

```diff
@@ -0,0 +1,47 @@
+package com.mail.entities;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.FieldDefaults;
+import org.springframework.lang.NonNull;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@Getter
+@Setter
+@NoArgsConstructor
+public class TextMessage {
+    static int counter = 0;
+
+    int id;
+    String title;
+    @NonNull
```

Comment:
If you have annotation above variable I suggest to write it this way:
```
int id;
String title;

@NonNull
String from;

@NonNull
String to;

String message;
...
```

This way it is easier to read code.